src/entity/Bullet.java

```diff
@@ -15,6 +15,7 @@ public class Bullet extends Entity {
      * Speed of the bullet
      */
     private int speed;
+    private int damage = 1;
```

Comment:
```suggestion
    private int damage;
```
아래 Bullet 생성자에서 damage를 입력받아 저장하기 때문에 초기값을 설정하지 않아도 될 것 같습니다. 추가로 변수에 대한 주석도 달아주세요. 없어도 상관은 없는데 코드 통일성을 위해 주석이 있으면 좋을 것 같습니다.